qhub/cli/_init.py

```diff
@@ -0,0 +1,496 @@
+            qmark=qmark,
+        ).ask()
+
+        repo_name = questionary.text(
+            f"And what will the name of this repository be? ({repo_url.format(git_provider=git_provider, org_name=org_name, repo_name='<repo-name>')})",
+            qmark=qmark,
+        ).ask()
+
+        inputs.repository = repo_url.format(
+            git_provider=git_provider, org_name=org_name, repo_name=repo_name
+        )
+
+        if git_provider == "github.com":
+            inputs.repository_auto_provision = questionary.confirm(
+                f"Would you like the following git repository to be automatically created: {inputs.repository}?",
+                default=False,
+                qmark=qmark,
+            ).ask()
+
+        # TODO: create `check_repository_creds` function
+        if not disable_checks:
+            pass
+
+        if git_provider == "github.com":
+            inputs.ci_provider = CiEnum.github_actions.value.lower()
+        elif git_provider == "gitlab.com":
+            inputs.ci_provider = CiEnum.gitlab_ci.value.lower()
+
+    # SSL CERTIFICATE
+    rich.print(
+        (
+            "\n 🪴  This next section is [italic]optional[/italic] but recommended. If you want your Nebari domain to use a Let's Encrypt SSL certificate, "
+            "all we need is an email address from you.\n\n"
+        )
+    )
+    ssl_cert = questionary.confirm(
+        "Would you like to add a Let's Encrypt SSL certificate to your domain?",
+        default=False,
+        qmark=qmark,
+    ).ask()
+
+    if ssl_cert:
+        inputs.ssl_cert_email = questionary.text(
+            "Which email address should Let's Encrypt associate the certificate with?",
+            qmark=qmark,
+        ).ask()
+
+    # ADVANCED FEATURES
+    rich.print(
+        (
+            "\n 🪴  This next section is [italic]optional[/italic] and includes advanced configuration changes. "
+            "⚠️ caution is advised!\n\n"
+        )
+    )
+    if questionary.confirm(
+        "Would you like to make advanced configuration changes?",
+        default=False,
+        qmark=qmark,
+    ).ask():
+
+        # NAMESPACE
+        inputs.namespace = questionary.text(
+            "What namespace would like to use?",
+            default=inputs.namespace,
+            qmark=qmark,
+        ).ask()
+
+        # TERRAFORM STATE
+        inputs.terraform_state = questionary.select(
+            "Where should the Terraform State be provisioned?",
+            choices=enum_to_list(TerraformStateEnum),
+            qmark=qmark,
+        ).ask()
+
+        # KUBERNETES VERSION
+        inputs.kubernetes_version = questionary.text(
+            "Which Kubernetes version would you like to use?",
+            qmark=qmark,
+        ).ask()
+
+    handle_init(inputs)
+
+    rich.print(
+        (
+            "Congratulations, you have generated the all important [purple]nebari-config.yaml[/purple] file 🎉\n\n"
```

Comment:
I think it'll be useful to highlight this first line because I almost missed it. Perhaps we can make it bold? It'll also be nice to add a new-line _before_ this statement to drive attention to it: 

![Screenshot 2022-09-29 at 10 51 14 PM](https://user-images.githubusercontent.com/33131404/193100240-1ab78df5-995e-4be5-985e-89bbc99cac79.png)

Reply:
Yeah, that's a good point. Making this change now.